### PR TITLE
IPD 15: Add more detail

### PR DESCRIPTION
**Pull request by pfmooney**
Adding more detail to the bhyve upstreaming proposal.

cc: @hrosenfeld @citrus-it

### CODE REVIEW

**Comment by pfmooney** [comment]
Updated to include Han's bits along with some formatting and the bootrom notes.

File: ipd/0015/README.md [hunk 1]
```diff
@@ -67,38 +67,38 @@ issue.
    for bhyve since it uses a dynamic sdev plugin to manage entries within
    /dev/vmm/
 
-1. [OS-6549](https://smartos.org/bugview/OS-6549) vmm segment driver
+2. [OS-6549](https://smartos.org/bugview/OS-6549) vmm segment driver
```
Comment (citrus-it) [inline ipd/0015/README.md:70]: Can you leave these all as `1.` ? Markdown takes care of numbering when it's rendered and it's much easier to add/remove/re-order things this way. (just for this list of pre-reqs, not the main section titles)

**Review by citrus-it — APPROVED** [review]

File: ipd/0015/README.md (hunk 1 above)
Comment (pfmooney) [reply ipd/0015/README.md:70]: I'm a fan of the markdown "rendering" roughly the same in the terminal as it does on github.

Comment (citrus-it) [reply ipd/0015/README.md:70]: Fair enough.

**Pull request merged**